content/operate/kubernetes/re-clusters/delete-custom-resources.md

```diff
@@ -11,6 +11,17 @@ linkTitle: Delete custom resources
 weight: 70
 ---
 
+{{< warning >}}
+Use graceful deletion methods
+
+When deleting Redis Enterprise resources or pods, try to avoid use `kubectl delete --force` or similar force deletion commands, as these can lead to:
```

Comment:
I'm wondering whether we should actually say that, because:
1. `--force` flag doesn't have any effect on custom resources - it doesn't change the deletion behavior. In fact, I'm not aware of any k8s resource besides Pods that support non-graceful deletion. The k8s documentation vaguely say that _"only a subset of resources support graceful deletion"_, but I believe this comes down to... pods only.
2. Regardless, users don't need to, and typically shouldn't, delete operator-managed resources on their own. They should only delete custom resources that they've created (REC, REDB, ...), and the operator should take care of deleting related/managed resources that it created. Particularly, users shouldn't be deleting REC pods directly.. this typically leads to problems.